t/test-lib-functions.sh

```diff
@@ -933,6 +933,14 @@ perl () {
 	command "$PERL_PATH" "$@" 2>&7
 } 7>&2 2>&4
 
+mswin32_perl () {
+	local GITPERLLIB="$GITPERLLIB"
+	if test_have_prereq CYGWIN || test_have_prereq MINGW; then
+		GITPERLLIB="$(cygpath -w -p "$GITPERLLIB")"
```

Comment:
Does that _really_ work? There are native `.dll` files that some of the Perl modules want to load, and I am _rather_ certain that they can be used _only_ by `/usr/bin/perl`. It _has_ to be an MSYS Perl. It even has to match the precise version, otherwise it won't load.